src/ol/style/IconImage.js

```diff
@@ -8,7 +8,7 @@ import ImageState from '../ImageState.js';
 import {asString} from '../color.js';
 import {createCanvasContext2D} from '../dom.js';
 import {shared as iconImageCache} from './IconImageCache.js';
-import {listenImage} from '../Image.js';
+import {load} from '../Image.js';
```

Comment:
Wouldn't `decodeFallback` be slightly more efficient, while still keeping SVGs scaleable?